src/main/java/com/google/cloud/tools/appengine/operations/Gcloud.java

```diff
@@ -247,6 +252,15 @@ public Builder setShowStructuredLogs(String showStructuredLogs) {
       return this;
     }
 
+    /**
+     * Sets the verbosity of output for gcloud. @see <a
+     * href="https://cloud.google.com/sdk/gcloud/reference#--verbosity">gcloud docs</a>
+     */
+    public Builder setVerbosity(String verbosity) {
+      this.verbosity = verbosity;
```

Comment:
This needs an enum or a check on the allowed values.

Reply:
I thought about that, but gcloud will just spit out the error.

Reply:
Failing earlier is better. Not failing at all because an enum prevents passing an incorrect value is better still.